python/arcticdb/util/environment_setup.py

```diff
@@ -742,11 +1068,45 @@ def test_setup_versions_and_snapshots(cls):
     	#Delete-setup-all-check
         cls.delete_test_store(setup)
         setup.setup_environment()
+        setup.logger().info(setup.get_arctic_client_persistent().list_libraries())
         assert setup.check_ok()
 
         #Changing parameters should trigger not ok for setup
         setup.set_with_metadata_for_each_version()
         setup.set_with_snapshot_for_each_version()
         setup.set_params([2, 3])
         assert not setup.check_ok()
+
+    @classmethod
+    def test_setup_multiple_libs_with_symbols0(cls):
+        params = [[2, 3], [4, 5]]
+        cols = 12
+        setup = (SetupMultipleLibraries(storage=Storage.LMDB, 
+                                                      prefix="BASIC_BATCH")
+                   .set_params(params) # For test purposes
+                   .set_number_columns(cols)
+                   ).set_test_mode()
+        assert setup.is_test_mode()            
         
+        cls.delete_test_store(setup)
+
+        assert not setup.check_ok()
+        setup.setup_environment()
+        #assert setup.check_ok()
+
+        ac = setup.get_arctic_client_persistent()
+        libs = ac.list_libraries()
+        setup.logger().info(f"LIBS {libs}")
+        assert len(libs) == len(params[0]) # the symbol list
+
+        for num_syms in params[0]: # the symbols list
+            ln = setup.get_library_name(LibraryType.PERSISTENT, num_syms)
+            lib = ac.get_library(ln)
+            symbols_list = lib.list_symbols()
+            setup.logger().info(symbols_list)
+            for rows in params[1]: # the symbols list
+                setup.get_symbol_name(num_syms, rows, cols)
+                assert setup.has_library(num_syms) 
+                lib = setup.get_library(num_syms)
+                symbol = setup.get_symbol_name(num_syms - 1, rows, cols)
+                assert lib.read(symbol).data is not None
```

Comment:
This assertion seems quite slow and expensive for what it tells us. If you have to have it, you should at least call `head` rather than reading the entire symbol